lib/writers/lvgl/lv_font.js

```diff
@@ -21,6 +21,14 @@ class LvFont extends Font {
       this.font_name = path.basename(options.output, ext);
     }
 
+    if (options.lv_fallback) {
+      this.fallback = '&' + options.lv_fallback;
+      this.fallback_declaration = 'const lv_font_t ' + options.lv_fallback + ';\n';
```

Comment:
I just noticed, this should be `extern ...` if anything.

I should probably also explain (in the commit message) why I added this declaration at all: It is so that a font can have a fallback that is not declared in LVGL's own headers.

Reply:
done